### PR TITLE
Hotfix #2 for minerva deployment

### PR DESCRIPTION
**Pull request by chrissank**

### CODE REVIEW

**Comment by jacobdeery** [comment]
Merge conflicts here - can you rebase on master?

File: functions/scheduled/events.js [hunk 1]
```diff
@@ -131,7 +131,9 @@ module.exports.parseDescription = async function (summary, description, channelI
         }
     }
 
-    parameters.extra = lines[5] === undefined ? "" : lines[5];
+    if (lines[5] !== undefined || lines[5] !== "") {
```
Comment (jacobdeery) [inline functions/scheduled/events.js:134]: This condition will always be true. Should it be `&&` instead of `||`?

Comment (chrissank) [reply functions/scheduled/events.js]: Should just be not undefined. Fixed.

**Comment by chrissank** [comment]
I'm satisfied with the current state, if you approve it we can merge before 130 today

File: functions/scheduled/events.js [hunk 2]
```diff
@@ -29,19 +29,19 @@ module.exports.checkForEvents = async function () {
 
             const message = await this.generateMessage(event, parameters, timeDifference, isEventSoon, startTimeDate);
 
-            await slack_handler.postMessageToChannel((parameters.alert_type === "alert-main-channel" ? "<!channel>\n" : "") + message, parameters.main_channel);
+            await slack_handler.postMessageToChannel((parameters.alert_type === "alert-main-channel" ? "<!channel>\n" : "") + message, parameters.main_channe, true);
```
Comment (jacobdeery) [inline functions/scheduled/events.js:32]: Intentional change (main_channel to main_channe)?

Comment (chrissank) [reply functions/scheduled/events.js]: Unintentional, fixed. Good eye, definitely slipped past me

**Pull request merged**